test/e2e/kubevirt.go

```diff
@@ -1267,8 +1278,11 @@ passwd:
 
 			if td.topology == "localnet" {
 				By("setting up the localnet underlay")
-				nodes := ovsPods(clientSet)
-				Expect(nodes).NotTo(BeEmpty())
+				err, nodes := ovsPods(clientSet)
+				Expect(err).To(Succeed())
```

Comment:
Nit: when checking the error instead of the function directlly it reads better like "this error should not ocurr" doing 
```suggestion
				Expect(err).ToNot(HaveOcurred)
```

Reply:
Fixed